libs/build_metplus.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash                                                                                                                              
+
+set -eux
+
+name="METplus"
+version=${1:-${STACK_metplus_version}}
+
+cd  ${HPC_STACK_ROOT}/${PKGDIR:-"pkg"}
+software="v"$version
+pkg_name=$name-$version
+met_version=${1:-${STACK_metplus_met_version}}
+url="https://github.com/dtcenter/METplus/archive/$software.tar.gz"
```

Comment:
Please use `git clone` if possible.

Reply:
See above comment addressing this issue of git clone vs. wget.